src/components/GamesContainer/GameCard/GameCard.module.css

```diff
@@ -0,0 +1,9 @@
+.card {
+    background-color: #303030;
+    border-radius: 12px;
+    width: 30%;
+    min-width: 320px;
```

Comment:
Please remember to use rem (for which we have a base font-size of 10px) instead of px for sizes.